Hublsoft.Net.LoginLogout.DataAccess/RegisteredUsersRepository.cs

```diff
@@ -105,5 +105,65 @@ public async Task IncrementFailedLoginAttemptsAsync(int id)
                 await con.CloseAsync();
             }
         }
+
+        public async Task<int> GetFailedLoginAttempsAsync(int id)
+        {
+            if (id == 0)
+            {
+                throw new Exception("Id cannot be 0");
+            }
+
+            var attempCount = 0;
+
+            try
+            {
+                using (var con = new MySqlConnection(ConnectionString))
+                {
+                    await con.OpenAsync();
+
+                    var stm = $"SELECT FailedLoginAttempts FROM RegisteredUsers WHERE Id = {id} ;";
+
+                    using (var cmd = new MySqlCommand(stm, con))
+                    {
+                        attempCount = (int)cmd.ExecuteScalar();
+                    }
+
+                    await con.CloseAsync();
+                }
+            }
+            catch
```

Comment:
the empty catch statement will hide any exceptions, always best to allow unhandled exceptions to bubble up and handle in the controller (I realise there is no exception handling in the controller yet or logging for that matter). Please remove the try ... catch

Reply:
Oooo yes that wouldn't be good. I'll remove this. Something we might want to think about for a our app is putting in some basic logging. Then we can pipe that into something sort of log stash.

Reply:
Yes, already in the Todo list in the readme.md